packages/packages-groonga-org-package-task.rb

```diff
@@ -40,6 +41,14 @@ def detect_release_time
     end
   end
 
+  def define_source_task
+    desc "Download the latest source archive"
+    task :source do
+      source_archive_url = built_package_url(nil, @archive_base_name)
+      download(source_archive_url, "..")
+    end
```

Comment:
Can we rewrite `define_archive_task` instead?

Reply:
Adressed. (Also changed the description of this PR.)

Reply:
We need to update title too.